.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,21 @@
-<!--
-  - Thanks for submitting code to Oppia! Please fill out the following as part of
-  - your pull request so we can review your code more easily.
-  -->
-
 ## Explanation
-<!--
-  - Explain what your PR does. If this PR fixes an existing bug, please include
-  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
-  - when this PR is merged.
-  -->
 
-## Checklist
-- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
-- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
+This PR fixes #[fill_in_number_here].
+
+This PR does the following and why...
+
+[Delete this and fill in your response here]
+
+## Essential Checklist
+
 - [ ] The linter/Karma presubmit checks have passed.
-  - These should run automatically, but if not, you can manually trigger them locally using `python -m scripts.pre_commit_linter` and `python -m scripts.run_frontend_tests`.
+- [ ] Allow edits from maintainers is checked, see [this](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for how to enable.
+  - This lets reviewers restart your CircleCI tests for you.
 - [ ] The PR is made from a branch that's **not** called "develop".
-- [ ] The PR has an appropriate "PR CHANGELOG: ..." label (If you are unsure of which label to add, ask the reviewers for guidance).
-- [ ] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
-- [ ] The PR addresses the points mentioned in the codeowner checks for the files/folders changed. (See the [codeowner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).)
-- [ ] The PR is **assigned** to an appropriate reviewer.
-  - If you're a new contributor, please ask on [Gitter](https://gitter.im/oppia/oppia-chat) for someone to assign a reviewer and don't tick this checkbox.
-  - If you're not sure who the appropriate reviewer is, please assign to the issue's "owner" -- see the "talk-to" label on the issue. Do not only request the review but also add the reviewer as an assignee.
+
+## PR Pointers
+
+- Oppiabot will notify you when you don't add a PR_CHANGELOG label, if you are unable do to so, please tag a codeowner.
+- For what code owners will expect, see the [codeowner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).
+- Linters and reviewers will look for conventions in the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
```

Comment:
Suggest changing to: Make sure your PR follows conventions in the style guide, otherwise this will lead to review delays.

(Don't want to imply that the linters and reviewers will do the work for them, and that the PR author doesn't need to worry about it.)

Reply:
done!